README.md

```diff
@@ -1,2 +1,37 @@
-# phenotype
-ruby gem for versioning
+# Phenotype
+ruby gem for versioning. Current supports Sinatra and other lightweight rack frameworks.
+
+## Versioning an Application
+Currently, versioning only supports blocks so you'll have to wrap your application like so.
+
+```ruby
+  app = Phenotype::VersionedApp(strategies: [Phenotype::HeaderStrategy.new, Phenotype::ParamStrategy.new])
+  app.add_version(version: 'v1', default: true) do |env|
+    MyApp::V1.new().call(env)
+  end
+
+  run app
+  
+```
+
+Note that for Sinatra apps you can create a cascading API using `use` in a Sinatra base
+
+```ruby
+class V2 < Sinatra::Base
+  use V2Routes
+  use V1Routes
```

Comment:
Does that mean that calling a v2 route that doesn't exist in v2 will fall through to the v1 route?